bundles/org.openhab.binding.mystrom/README.md

```diff
@@ -24,13 +27,36 @@ The following parameters are valid for all thing types:
 | hostname  | string  | yes      | localhost          | The IP address or hostname of the myStrom smart plug                       |
 | refresh   | integer | no       | 10                 | Poll interval in seconds. Increase this if you encounter connection errors |
 
+## Properties
+
+In addition to the configuration a myStrom thing has the following properties.
+The properties are updated during initialize. Disabling/enabling the thing can be used to update the properties.
```

Comment:
```suggestion
The properties are updated during initialize.
Disabling/enabling the thing can be used to update the properties.
```

Start each sentence in the docimentation on a new line. This makes reviewing later documentation changes easier.